### DISCUSSION TITLE
Advantages of osu!(lazer) on Steam

### DISCUSSION THREAD

**Discussion by Pacific6938** [ppy/osu#22694] [Ideas]
We suggest that osu!lazer should be made available on Steam. The game has now become stable and polished, and we believe it would greatly benefit from the accessibility that the Steam platform provides.

By adding the Steam Early Access tag, players would understand that the game is still in development. The ease of use provided by Steam would make the installation process simpler and more intuitive for new players.

Moreover, bringing osu!lazer to Steam would make it easier to play on devices such as the Steam Deck. The added support for controllers through Steam XInput would make the game more accessible to players who prefer using controllers over traditional mouse and keyboard controls.

In addition, the changelog for the game could also be made available on Steam, since people are familiar with the Steam changelog system. Additionally, there could be guides on the Steam community page to help new players get started with the game, such as a guide to move the mouse or set up the trackpad or learn streams.

By making osu!lazer available on Steam, we believe the game would be exposed to a much larger audience than it currently has, potentially expanding the player base and community.

This text was written and suggested by the osu!lazer Community members Me and PaperTobi.

**Comment by Pacific6938** [comment]
Also, I think bringing osu!lazer to Steam would greatly improve the update experience for Linux users. The current way of updating on Linux is quite complicated, and if you want to update manually, it takes time because you have to replace the AppImage with the new one. By making the game available on Steam, Linux users would be able to receive automatic updates like the Windows users, which would make the process much simpler and more convenient for them.

**Comment by Axilotl17** [comment]
#18074

Is there a great enough demand for it? 
additionally, updates for linux will come eventually, and I'm pretty sure the [zsync file](https://github.com/ppy/osu/releases/download/2023.207.0/osu.AppImage.zsync) is how one auto-updates lazer on linux currently.

**Reply by Pacific6938** [reply]
We have listed several reasons why we believe that osu should be available on Steam. While demand is a significant factor, we also believe that exposing the game to a larger audience is another compelling point.
Although zsync is a potential auto-update solution, I have spent over an hour trying to understand how it works and still do not have any idea how it works.

**Reply by bdach** [reply]
Please redirect your support to the aforementioned discussion thread so that all support for doing this is concentrated in one place. If it gets enough traction (gauged via upvotes/reactions) with time we can reconsider.

**Reply by Axilotl17** [reply]
@Schweeeeeeeeeeeeeeee i'll take a look and see if i can get zsync working. I'm also curious.

**Reply by Pacific6938** [reply]
> @Schweeeeeeeeeeeeeeee i'll take a look and see if i can get zsync working. I'm also curious. 

I'd love to hear back from you and how you did it

**Reply by Axilotl17** [reply]
@Schweeeeeeeeeeeeeeee https://github.com/AppImageCommunity/zsync2

`zsync2 https://github.com/ppy/osu/releases/download/2023.207.0/osu.AppImage.zsync` in the directory with the previous `osu.AppImage` file.

**Comment by darkralts** [comment]
i think steam would be cool, would make things easier and more people would find the game probably

**Comment by ItsAyaya** [comment]
I think osu! available on Steam could make more people around the world know about this game and play it. By adding Steam Overlay and Steam Rich presence, Steam friends could know what beatmap their friends are currently playing.
Also, when a friend wants to play with me in multiplayer mode, all he has to do is just chat with me on Steam Overlay and invite me to his room. It's quite convenient.

**Comment by Temanor** [comment]
I don't think a steam release for stable would make sense, but with the move to lazer, I think this would be great.

**Comment by mixalisnew** [comment]
dodnt you have to pay something like monthly to have a game on steam?

**Comment by Tenexxt** [comment]
Lazer on steam would be so cool. My friends would finally know just how much of a no-life I am.

**Reply by ziv-vy** [reply]
Just add osu! as a non-steam game. The boot time will increase by a few seconds but that's it - your friends **will** know :)

**Reply by Tenexxt** [reply]
Didn't know that adding the game this way will also alert people of what I'm playing correctly, will do that!

**Comment by ziv-vy** [comment]
Duplicate of #18074 - no need for two threads.

**Comment by rvnexe** [comment]
# BRO FORGOT TO MENTION "SHARING SKINS ON WORKSHOP!!!"

**Reply by rvnexe** [reply]
- Adding steam friends on osu! account
- Inviting steam friends to play multiplayer
- Rich presence of "playing osu!" in steam friends

**Comment by genskyff** [comment]
I think osu!lazer would be perfect for Steam release. While stable might not need it, lazer has some great advantages:
Why it makes sense:

- The UI/UX is polished and visually impressive enough for Steam
- After years of development, it's definitely playable even if not 100% stable yet
- Steam's auto-updates would be seamless
- Much better Linux/Steam Deck support
- Steam Workshop for skins (like Quaver does) - centralized skin sharing would be amazing
- Steam achievements would give the current achievement system more visibility and motivation
- Huge opportunity to reach new players beyond the existing community

The game has reached a quality level that would really shine on Steam. Hope the dev team considers this seriously!

**Reply by darkralts** [reply]
perhaps steam could be used as an official release for when lazer becomes the main client

**Reply by darkralts** [reply]
no means against an early access tho :o
